From c67b37a75c6b76da744853a24d029f8b5fde9177 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 15 Jan 2014 09:14:32 -0500
Subject: [PATCH 05/14] audio: Lower default wakeup rate to 100 times / second

Message-id: <f7a7da0d506fd9969e30a4a164dbfe5146bba766.1389776721.git.mrezanin@redhat.com>
Patchwork-id: 56720
O-Subject: [RHEL7 qemu-kvm PATCH 2/3] audio: Lower default wakeup rate to 100 times / second
Bugzilla: 1017636
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

From: Hans de Goede <hdegoede@redhat.com>

This is more then plenty to keep audio card fifos filles / emptied.

This drops host cpu-load for audio playback inside a linux vm from
13% to 9%.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit 40a814b0b1789b94c483190b3208729b5182e5bd)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 audio/audio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 audio/audio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/audio/audio.c b/audio/audio.c
index ebf7d3f..33b3fb6 100644
--- a/audio/audio.c
+++ b/audio/audio.c
@@ -95,7 +95,7 @@ static struct {
         }
     },
 
-    .period = { .hertz = 250 },
+    .period = { .hertz = 100 },
     .plive = 0,
     .log_to_monitor = 0,
     .try_poll_in = 1,
-- 
1.8.3.1

